platform-ajax/src/test/java/com/softicar/platform/ajax/input/auto/complete/entity/AbstractAjaxAutoCompleteEntityTest.java

```diff
@@ -344,17 +347,6 @@ public IndicatorExpectationSetter expectValuesNone() {
 			return expectValues(null);
 		}
 
-		public ClientValueExpectationSetter expectServerValue(AjaxTestEntity serverValue) {
```

Comment:
These methods allowed for mixing up the call order on the Asserter, for no good reason. Removed.

Reply:
best code is no code